src/snowflake/connector/connection.py

```diff
@@ -235,7 +235,11 @@ def _get_private_bytes_from_file(
     "connection_diag_whitelist_path": (
         None,
         (type(None), str),
-    ),  # Path to connection diag whitelist json
+    ),  # Path to connection diag whitelist json - Deprecated remove in future
```

Comment:
I have one question and one request:

- is `connection_diag_whitelist_path` only internally used by us (not external customer facing)? if this api keeps change, shall we consider making it private by naming like `_connection_diag_whitelist_path` (add a underscore at the front)
- can you create a JIRA for the future deprecation?

Reply:
connection_diag_whitelist_path and connection_diag_allowlist_path are the public parameter names for a path to the allowlist/whitelist if they need to provide one.  So it should stay public, right?

Filed a jira for future deprecation.

Reply:
I think these are resolved.